cap/modules/deposit/minters.py

```diff
@@ -30,13 +30,27 @@
 
 from invenio_pidstore.models import PersistentIdentifier, PIDStatus
 
+from cap.modules.schemas.resolvers import resolve_schema_by_url
+from .utils import generate_auto_incremental_pid
+
 
 def cap_deposit_minter(record_uuid, data):
     """Mint deposit's identifier."""
     try:
         pid_value = data['_deposit']['id']
     except KeyError:
-        pid_value = uuid.uuid4().hex
+        pid_value = None
```

Comment:
if we need to check the experiment, we can use the `_experiment` property, like it is done on `cap_record_minter`